flutter_vlc_player/android/src/main/java/software/solid/fluttervlcplayer/FlutterVlcPlayerPlugin.java

```diff
@@ -56,22 +56,24 @@ public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {
     @RequiresApi(api = Build.VERSION_CODES.N)
     @Override
     public void onAttachedToActivity(@NonNull ActivityPluginBinding binding) {
-        final FlutterInjector injector = FlutterInjector.instance();
-        //
-        flutterVlcPlayerFactory =
-                new FlutterVlcPlayerFactory(
-                        flutterPluginBinding.getBinaryMessenger(),
-                        flutterPluginBinding.getTextureRegistry(),
-                        injector.flutterLoader()::getLookupKeyForAsset,
-                        injector.flutterLoader()::getLookupKeyForAsset
-                );
-        flutterPluginBinding
-                .getPlatformViewRegistry()
-                .registerViewFactory(
-                        VIEW_TYPE,
-                        flutterVlcPlayerFactory
-                );
-        //
+        if(flutterVlcPlayerFactory == null) {
```

Comment:
Nitpick: missing space after `if`.

```suggestion
        if (flutterVlcPlayerFactory == null) {
```

Reply:
Done